html-forms/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label for="email">Email</label><br />
+      <input type="email" id="email" name="email" /><br />
+      <label for="simple">Why is coding cool?</label><br />
+      <select id="simple" name="simple">
+        <option>Select an option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option></select
+      ><br /><br />
+      <label>Favorite Language (choose one)</label><br /><br />
```

Comment:
Since this `label` isn't associated with any form control element, this should just be a `p`